### PR TITLE
Nexus-20700: conan proxy official support merge master

### PR DESCRIPTION
**Pull request by igorudovika**
Nexus-20700: Merged master in Conan feature branch

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+language: java
```
Comment (aornatovskyy) [inline .travis.yml]: we don't use Travis at all. We can remove it.

File: Dockerfile [hunk 2]
```diff
@@ -1,15 +1,15 @@
 ARG NEXUS_VERSION=3.15.1
 
 FROM maven:3-jdk-8-alpine AS build
-ARG NEXUS_VERSION=3.15.1
+ARG NEXUS_VERSION=3.18.1
```
Comment (aornatovskyy) [inline Dockerfile]: why 3.18? and not latest one? 
But in general, after we include conan we will update this version also

Comment (igorudovika) [reply Dockerfile]: I think it should be 3.18 as was lust update in master. This is for old versions support. We did not cover old Conan version on top of 3.19 - 3.21. In 3.22 that we are working this is redundant

File: .travis.yml (hunk 1 above)
Comment (igorudovika) [reply .travis.yml]: Fixed 22ef989e35d08408dc00e5aef18389dfff15b8cd

Comment (aornatovskyy) [reply .travis.yml]: LGTM

File: Dockerfile (hunk 2 above)
Comment (aornatovskyy) [reply Dockerfile]: we will update this version later.

**Pull request merged**